spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -49,22 +51,30 @@ def run
         cookie.name.start_with?("myacinfo") || cookie.name == "dqsid" || cookie.name.start_with?("DES")
       end
 
-      yaml = cookies.to_yaml.gsub("\n", "\\n")
+      @yaml = cookies.to_yaml.gsub("\n", "\\n")
 
       puts("---")
       puts("")
       puts("Pass the following via the FASTLANE_SESSION environment variable:")
-      puts(yaml.cyan.underline)
+      puts(@yaml.cyan.underline)
       puts("")
       puts("")
       puts("Example:")
-      puts("export FASTLANE_SESSION='#{yaml}'".cyan.underline)
+      puts("export FASTLANE_SESSION='#{@yaml}'".cyan.underline)
 
-      if mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true)
-        require 'open3'
-        Open3.popen3('pbcopy') { |input, _, _| input << yaml }
-        puts("Successfully copied text into your clipboard 🎨".green)
+      if @copy_to_clipboard == false
+        puts("Skipped asking to copy the session string into your clipboard ⏭️".green)
+      elsif @copy_to_clipboard || (mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true))
+        FastlaneCore::Clipboard.copy(content: @yaml)
+        puts("Successfully copied the session string into your clipboard 🎨".green)
       end
+
+      return self
```

Comment:
Can this just return `@yaml`? 😇 This is already running on an instance so I don’t think returning `self` is as useful as returning the actual value? 🤔 

This also gets rid of the need for the `session_string` addition below 🤷‍♂️ 

cc: @rogerluan @return-main

Reply:
I'd be okay with that, but by returning `self` we can future-proof the lane in case we create new methods.

Reply:
I share the same thoughts as @return-main  ☝️ 
Not sure how/if this class plans to be changed much in the future, but it's a possibility. It would make it possible to create new methods similar to how `session_string` works, without worrying about retrocompatibility.

Reply:
I’m fine with that 😇  We should be good to go then